tasks.py

```diff
@@ -196,21 +285,31 @@ def _update_version(version_: Version | str) -> None:
 def _version_bump(ctx: Context, pre: bool = False, standard: bool = False) -> None:
     """Bump project version and release to pypi."""
     local_version = _get_local_version()
-    print(f"local: \t\t{local_version}")
-    latest_version = _get_latest_version()
-    print(f"pypi latest: \t{latest_version}")
+    print(f"local was: \t\t\t{local_version}")
+    latest_pre_release_version, latest_release_version = _get_latest_versions()
+    print(f"pypi latest release: \t\t{latest_release_version}")
+    print(f"pypi latest pre-release: \t{latest_pre_release_version}")
 
+    release_type = "standard release" if standard else "pre-release"
     if standard:
         pre = False
     elif not pre:
         # if not explicitly set to standard release, default to pre-release
         pre = True
 
-    print("\n  bumping version ...", end=" ")
-    new_version = bump_version(local_version, pre_release=pre)
+    print(f"\nbumping version ({release_type}) ...", end=" ")
+
+    new_version = bump_version(
+        pre_release=pre,
+        latest_version=latest_release_version,
+        latest_pre_release_version=latest_pre_release_version,
+        current_date=datetime.date.today().strftime("%Y%m%d"),  # noqa: DTZ011 # timezone agnostic, local time OK.
```

Comment:
I think we should use timezone aware UTC time here. It totally eliminates an entire class of problems relating to multiple releases in a short timespan that go back and forth between a dateline.

Reply:
Good point - I was mainly worried about someone releasing later in the day and seeing the date be the next day because it is "tomorrow" in UTC. But you're right - I agree that standardizing is better so I'll make this change.

Reply:
To handle the "tomorrow" issue ☝️ I was going to set to ET but didn't want to add another dependency just for that.